push-notifications/basics/ios.md

```diff
@@ -202,6 +202,34 @@ operation?.sendCompletionHandler = { (deviceIDs, error) in
 }
 ```
 
+## Unregistering device
+
+When a device is registered, it is associated with the user. Therefore unregister is needed when the user is no longer associated with the device, for example when the user logouts.
```

Comment:
@b123400 I have some comments for this sentence:

- Better break the sentence for the 80-character line limit, and
- "it is associated with the authenticated user"
- "Therefore it is necessary to unregister the device when" (See notes below)
- "when the user logs out."

I believe "deregister" is more appropriate from a pure English standpoint, but given that "unregister" is more popular we will stick with "unregister". However "unregistering" or "unregistration" does not sound natural to me